content/en/docs/operations/troubleshooting/linstor-database.md

```
@@ -44,141 +54,349 @@ If you see that something could not delete properly, it's better to investigate
 ```
 
-{{% alert color="warning" %}}
-:warning: DESTRUCTIVE ACTION!
+Find all resources for that resource and node:
+```bash
+chmod +x linstor-find-db.sh
+./linstor-find-db.sh resource_name=PVC-91C1486F-CFE9-41E2-80E1-86477B187F2D node_name=GLD-CSXHK-006
+```
+
+Example output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)